src/main/scala/nl.knaw.dans.easy.deposit/package.scala

```diff
@@ -70,7 +70,7 @@ package object deposit {
     }
   }
 
-  implicit class StringExtensions(val s: String) {
+  implicit class StringExtensions(val s: String) extends AnyVal {
```

Comment:
Could you add a `// TODO` to this implicit class, saying that this is no longer needed once we install better-files v3.6 ([link 1](https://github.com/pathikrit/better-files/blob/master/CHANGES.md#v360), [link 2](https://github.com/pathikrit/better-files/commit/5afb5f1ac58b248582e5cffcd8f32ebb2d91cd83)).

I'll probably get to that upgrade early next week, so stay tuned!